src/applications/disability-benefits/all-claims/tests/config/781.unit.spec.js

```diff
@@ -161,5 +161,17 @@ describe('781/781a incident form config iterators', () => {
         ).to.haveOwnProperty('type');
       }
     });
+    it('should return three secondary upload sources page config objects', () => {
+      const config = createFormConfig781a(PTSD_INCIDENT_ITERATION);
+
+      const testMultipleSecondaryUploadSourcesProps = () => {
```

Comment:
Can you remove the wrapping function here?  I think the test name is clear enough and I've been trying to clean these up